.dockerignore

```diff
@@ -0,0 +1,13 @@
+.*.un~
+*.log
+.env
+.env.template
+.env.test
+.git
+.github
+.gitignore
+.vagrant
+Vagrantfile
+.coverage
+.pytest_cache
+poetry.lock
```

Comment:
Your `poetry.lock` doesn't really want to be ignored - it should be copied in with your `pyproject.toml` to help closely version your exact dependencies so that you'll maintain better consistency

Reply:
Ah wasn't aware that's what this file was for, have updated, thanks Jack 🙂